core-services/prow/02_config/_plugins.yaml

```diff
@@ -7706,6 +7733,34 @@ plugins:
   redhat-developer/service-binding-operator:
     plugins:
     - approve
+  redhat-et/microshift:
+    plugins:
+    - assign
+    - blunderbuss
+    - bugzilla
+    - cat
+    - dog
+    - heart
+    - golint
+    - goose
+    - help
+    - hold
+    - jira
+    - label
+    - lgtm
+    - lifecycle
+    - override
+    - pony
+    - retitle
+    - shrug
+    - sigmention
+    - skip
+    - trigger
+    - verify-owners
+    - owners-label
+    - wip
+    - yuks
+    - approve
```

Comment:
also, u can remove most of these (I'd definitely keep the cat, dog and goose :) https://prow.k8s.io/plugins
these control GH comment triggers, you'll probably want 
- lgtm
- wip
- approve (?)